pkg/rpm/postupgrade

```diff
@@ -150,63 +150,81 @@ fi
 echo "Creating a backup of elasticsearch-serviced data in $HOST_ISVCS_DIR"
```

Comment:
I assume this should be deleted because I see the same two lines in the next if-condition